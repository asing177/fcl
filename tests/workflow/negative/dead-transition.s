// Dead transition transition `b -> end`
@initial
go1() { transitionTo(@a) }

@a
go2() { transitionTo(@terminal) }

@b
go3() { transitionTo(@terminal) }

