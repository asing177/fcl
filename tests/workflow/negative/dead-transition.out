Workflow soundness errors:
â€¢ Unreachable: 'transition {b} -> {terminal}'.